lib/setup.py

```diff
@@ -37,7 +37,7 @@
     "importlib-metadata>=1.4",
     "numpy",
     "packaging>=14.1",
-    "pandas<2,>=0.25",
+    "pandas==1.0.*",
```

Comment:
Should this be `pandas>=1.0, <2`? (Also, any concerns about "dropping" support for old Pandas?)